.github/workflows/publish-docs.yml

```
@@ -64,16 +66,22 @@ jobs:
+      - name: Verify docs directories
         run: |
           ls -la packages/sdk/docs/html
+          ls -la packages/actions/docs/html
           echo "Verifying documentation files exist..."
+      - name: Assemble Pages site
+        run: |
+          mkdir -p site/actions
+          cp -R packages/sdk/docs/html/. site/
+          cp -R packages/actions/docs/html/. site/actions/
       - name: Setup Pages
         uses: actions/configure-pages@v2
```

actionlint (1.7.12)

[error] the runner of "actions/configure-pages@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)